src/components/connect.js

```diff
@@ -70,30 +198,30 @@ export default function connect(mapPropsToRequestsToProps, options = {}) {
 
   function assignDefaults(mapping, parent) {
     return Object.assign(
+      { comparison: defaults.comparison },
       parent ? {
         comparison: parent.comparison
       } : {},
       {
-        method: 'GET',
-        credentials: 'same-origin',
-        redirect: 'follow',
-        meta: {}
+        andCatch: defaults.andCatch,
+        andThen: defaults.andThen,
+        catch: defaults.catch,
+        credentials: defaults.credentials,
+        force: defaults.force,
+        meta: {},
```

Comment:
Neither `meta` nor `value` are in defaults, mostly because I figured they were special cases already as they're only used for development/faking responses/ad-hoc custom responses in fetch block functions/etc.